File: ChurchAdminstrationsClient/app/member/member.js
(function () {
    'use strict';
    var getMember = 'member';
    

    angular.module('app').controller(getMember, ['common','datacontext', member]);
    
       


       
    
      

            /*  //Assign Values
              if ($scope.name) { var name = $scope.name; } else { var name = null; }
              if ($scope.username) { var username = $scope.username; } else { var username = null; }
              if ($scope.email) { var email = $scope.email; } else { var email = null; }
              if ($scope.dateofbirth) { var dateofbirth = $scope.dateofbirth; } else { var dateofbirth = null; }
              if ($scope.password) { var password = $scope.password; } else { var password = null; }
              if ($scope.confirmpassword == $scope.password) { var confirmpassword = $scope.confirmpassword; } else { var confirmpassword = null; }
              if ($scope.gender) { var gender = $scope.gender; } else { var gender = null; }
              if ($scope.mobileno) { var mobileno = $scope.mobileno; } else { var mobileno = null; }
              if ($scope.city) { var city = $scope.city; } else { var city = null; }
              if ($scope.occupation) { var occupation = $scope.occupation; } else { var occupation = null; }
              if ($scope.education) { var education = $scope.education; } else { var education = null; }
  
              if ($scope.group) { var group = $scope.group; } else { var group = null; }
              if ($scope.ministry) { var ministry = $scope.ministry; } else { var ministry = null; }
  
              if ($scope.membertype) { var membertype = $scope.membertype; } else { var membertype = null; }
              if ($scope.baptistdate) { var baptistdate = $scope.baptistdate; } else { var baptistdate = null; }
              if ($scope.address) { var address = $scope.address; } else { var address = null; }
              if ($scope.file) { var file = $scope.file; } else { var file = null; }
              if ($scope.maritalstatus) { var maritalstatus = $scope.maritalstatus; } else { var maritalstatus = null; }*/

         
   
     

    function member(common, datacontext) {
        var getLogFn = common.logger.getLogFn;
        var log = getLogFn(getMember);

        var vm = this;
        vm.members = [];
        vm.title = 'Member';

        activate();

        function activate() {
            var promises = [getMembers()];
            common.activateController(promises, getMember)
                .then(function () { log('Activated Member View'); });
        }

        function getMembers() {
            return datacontext.getMembers().then(function (data) {
                vm.members = data.$values;
                console.log(vm.members);
                return vm.members;

            });
        }
       
    }


   
    
})();

File: ChurchAdministrations/Models/Member.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Runtime.Serialization;
using System.Web;

namespace ChurchAdministrations.Models
{ 
    
    public class Member
    {

        public int MemberID { get; set; }



        public string Name { get; set; }
        public string Username { get; set; }
        
      
        public string Password { get; set; }

       
        public string ConfirmPassword { get; set; }


        public string Email { get; set; }

 
        public string Gender { get; set; }

        public string MobileNo { get; set; }


        [DataType(DataType.Date)]
        [DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]
        public DateTime DateOfBirth { get; set; }
        
        public string MaritalStatus { get; set; }

        public string MemberType { get; set; }
        public string Occupation { get; set; }

        public string Education { get; set; }

     
        


        [DataType(DataType.Date)]
        [DisplayFormat(DataFormatString = "{0:yyyy-MM-dd}", ApplyFormatInEditMode = true)]
        public DateTime BaptistDate { get; set; }

        public string City { get; set; }
       
        public string Address { get; set; }
        public string Image { get; set; }

        public int MinistryID { get; set; }

        public virtual Ministry Ministry { get; set; }
        public virtual Group Group { get; set; }

        public virtual ICollection<Pledge> Pledge { get; set; }
        public virtual ICollection<SellGift> SellGift { get; set; }
        public virtual ICollection<Accountant> Accountant { get; set; }

        public int GroupID { get; set; }

        



    }

    
}